google-cloud-tools-plugin/src/com/google/gct/idea/debugger/CloudDebuggerClient.java

```diff
@@ -46,8 +46,8 @@
  */
 public class CloudDebuggerClient {
 
-  private static final int LONG_CONNECTION_TIMEOUT_MS = 120 * 1000;
-  private static final int SHORT_CONNECTION_TIMEOUT_MS = 10 * 1000;
+  static final int LONG_CONNECTION_TIMEOUT_MS = 120 * 1000;
```

Comment:
In this case I think these may be used in other parts of the code later, so that's why I have not added the annotation. I actually think these could become configurable eventually by the user.